### PR TITLE
Update new_guide_setup.md

### PR DESCRIPTION
**Pull request by dockerymick**
No issue, just thought it would be helpful so that we can avoid the following error message:

Screen Shot 2021-07-19 at 3 37 13 PM

### CODE REVIEW

File: _pages/new_guide_setup.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ Only rename a folder/guide when absolutely needed as a result of product change.
    6. Click **Edit title**.
    7. Change the name and the URL fragment. Keep URL fragment in lowercase.
    8. Click **Rename** to save.
-   9. When links and files/folders are all changed, as well as titles, run a build.
+   9. Select the drop-downn error for the book and click **Edit Configuration**.
```
Comment (cadawson) [inline _pages/new_guide_setup.md]: downn>down
Is there a drop down for the error? I always just select the Down arrow icon for the book, > Edit configuration. Is that what you mean, or is there a better way?

**Review by cadawson — APPROVED** [review]
Thanks for updating this. One typo and one quick clarification, is all. Feel free to merge on.

File: _pages/new_guide_setup.md (hunk 1 above)
Comment (cadawson) [reply _pages/new_guide_setup.md]: Wait, maybe you mean arrow and typed error accidentally? That just occurred to me.

Comment (dockerymick) [reply _pages/new_guide_setup.md]: Thanks! when you click the arrow, a menu drops down so I would stick with drop-down arrow

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the reviews @swopebe @cadawson !

**Pull request merged**